chainforge/react-server/src/backend/queryTracker.ts

```diff
@@ -0,0 +1,54 @@
+export default class QueryTracker {
```

Comment:
Can you rename this to `QueryStopper` ? I'm finding that its name does not adequately reflect how it's used (it's currently just a way to check what queries to "stop" prematurely.)